gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala

```diff
@@ -67,7 +67,10 @@ private final class RunResultProcessor(configuration: GatlingConfiguration) {
 
   private def runStatus(assertionResults: List[AssertionResult]): StatusCode = {
     val consolidatedAssertionResult = assertionResults.foldLeft(true) { (isValid, assertionResult) =>
-      println(s"${assertionResult.message} : ${assertionResult.result}")
+      if (assertionResult.result)
+        println(s"${assertionResult.message} : ${assertionResult.result}")
+      else
+        println(s"${assertionResult.message} : ${assertionResult.result} (actual : ${assertionResult.actualValue.getOrElse(-1.0)})")
```

Comment:
IMO, `-1.0` is a magical value that doesn't convey any meaning.
If we don't have an actual value, I'd rather print `-` instead.

Reply:
Replaced `-1.0` with `-` per feedback.